src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -117,7 +117,7 @@ public function preLexComponents(string $input): string
     private function consumeComponentName(): string
     {
         $start = $this->position;
-        while ($this->position < $this->length && preg_match('/[A-Za-z0-9_]/', $this->input[$this->position])) {
+        while ($this->position < $this->length && preg_match('/[A-Za-z0-9_:]/', $this->input[$this->position])) {
```

Comment:
Should we future proof this by allowing `/`, `@` and `.`?

Reply:
and - ?

Reply:
`<twig:my-component/>`?

Reply:
yeah why not! We never know what people can bring 😅

Reply:
ha ok fine that was the first in your list! I missed it! Sorry about that

Reply:
(I edited the comment) 😁